src/examples/blinkm/blinkm.go

```diff
@@ -20,20 +20,18 @@ func main() {
 
 	count := 0
 	for {
-		machine.I2C0.WriteTo(0x09, []byte("n"))
```

Comment:
This change in examples/blinkm looks good, but should be done in a different PR. It appears to be unrelated to the nrf implementation.
In general, I prefer such examples in different PRs / commits, but when it is implemented for the first time (like on AVR) it's useful to have a real program to test it so that's why I was fine with it.

Reply:
Actually it is needed here. The AVR is forgiving enough that the extra STOP did not affect it, but it caused problems on the nrf. Changing this works on both.